service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/documents/ManageDocumentsController.java

```diff
@@ -320,14 +320,10 @@ public void handleSubmitted(@RequestBody CallbackRequest request) {
         CaseData caseData = getCaseData(request);
 
         DocumentUploaderType userType = getUploaderType(caseData.getId());
+        UserDetails userDetails = userService.getUserDetails();
 
-        if (this.featureToggleService.isNewDocumentUploadNotificationEnabled()
-            || (!DocumentUploaderType.SOLICITOR.equals(userType) && !DocumentUploaderType.BARRISTER.equals(userType))) {
-            UserDetails userDetails = userService.getUserDetails();
-
-            publishEvent(new FurtherEvidenceUploadedEvent(getCaseData(request),
-                getCaseDataBefore(request), userType, userDetails));
-        }
+        publishEvent(new FurtherEvidenceUploadedEvent(getCaseData(request),
```

Comment:
So we're removing this here and checking in the FurtherEvidenceUploadedEventHandler? Just wondering why? (sure there's a reason just trying to understand :) )

Reply:
because the FurtherEvidenceUploadedEvent is not limited to notification.

 I have added an event handling for creating a work allocation task so it shouldn't be checked here.